include/zephyr/posix/pthread.h

```diff
@@ -9,11 +9,11 @@
 
 #include <zephyr/kernel.h>
 #include <zephyr/wait_q.h>
-#include <zephyr/posix/time.h>
+#include <time.h>
 #include <zephyr/posix/unistd.h>
 #include "posix_types.h"
 #include "posix_sched.h"
-#include <zephyr/posix/pthread_key.h>
```

Comment:
this looks like a spurious change.

Reply:
It is, but isn't as well. Unrelated to time, so I'll restore here, but eventually I think this header should probably use double quotes like the other non-standard includes.